sunpy/map/tests/test_reproject_to.py

```diff
@@ -119,3 +120,16 @@ def test_return_footprint(aia171_test_map, hpc_header):
 def test_invalid_inputs(aia171_test_map, hpc_header):
     with pytest.raises(ValueError, match="The specified algorithm must be one of"):
         aia171_test_map.reproject_to(hpc_header, algorithm='something')
+
+def test_reproject_to_warn(aia171_test_map, hpc_header):
```

Comment:
Can we get a version of this test with using the context managers?

Reply:
yeah it can be added. But are we keeping both the tests? or change this one?